source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,35 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB, such as changes to a collection,
+database, or deployment, by using the :node-api:`watch()
+</Collection.html#watch>` method. ``watch()`` opens a :manual:`Change
+Stream</changeStreams/>`, that allows applications to watch for changes
+to data and react to those changes. After opening a ``Change Stream``,
+you can process the change events using the
+:node-api:`changeStream.next() </ChangeStream.html#next>` method.
```

Comment:
Nitpick: Here, I think you need to identify what `changeStream` is (an instance of a Change Stream) since it is not implied just by the name. You can refer to it as "the ``next()`` method on an instance of a Change Stream."

Reply:
Fixed to:
After opening a ``Change Stream``,
you can process the change events using the
`next()` method on an instance of a ``Change Stream``

Reply:
Perhaps you can omit the first part of the sentence to prevent repetition of a proper noun, or replace the second instance with "it".
At this point in the guide, it may not be clear that opening a Change Stream results in an instance since you're mentioning what it returns only on line 26 in the "Return" section.